lapis-docs/src/content/docs/maintainer-docs/concepts/caching.mdx

```diff
@@ -7,10 +7,17 @@ import CacheSizeEndpointLink from '../../../../components/CacheSizeEndpointLink.
 
 LAPIS uses Spring Boot's caching capabilities to cache aggregated, insertion and mutation queries.
 We decided to cache SILO actions that are relatively expensive to compute,
-but don't return much data (measured in size of the response body).
+but don't return much data (measured in size of the response body). Details and FASTA actions are expensive, but return
+potentially large amounts of data and are therefore not cached.
 
-Details and FASTA actions are expensive, but return potentially large amounts of data.
+Internally, we use soft references to allow the JVM garbage collector to remove entries from the cache if the memory is needed.
+However, per default the cache is not guaranteed to have a fixed memory size, increasing with each stored entry.
+The maintainer must ensure that enough memory is available to store the cache entries, or provide a limit to the
+heap size of the JVM (eg. `-Xmx4g`, or `-XX:+UseContainerSupport` when running LAPIS in docker )
+
+You can turn off caching by setting the `spring.cache.type` property to `none` in the `application.properties` file, or
```

Comment:
A maintainer will not modify the file, since it's already bundled in the Docker image. I think it's possible to provide a file as a maintainer, but I think in the other docs we say that providing the command line arguments is the way to go. -> Let's also just mention that here?